src/server/helpers/html.js

```diff
@@ -6,20 +6,51 @@ import { Provider } from 'react-redux';
 import { conf } from '../helpers/config';
 
 const GAID = conf.get('GOOGLE_ANALYTICS_ID');
+const OPTID = conf.get('GOOGLE_OPTIMIZE_ID');
+const GTMID = conf.get('GOOGLE_TAG_MANAGER_ID');
 
-const googleTagManager = GAID ?
+const googleOptimizePageHideCss = GAID && OPTID ?
+  <style
+    dangerouslySetInnerHTML={{ __html: `.async-hide { opacity: 0 !important}` }}
```

Comment:
I think linter is failing here as it complains about usage of template string `...` without any variables used in it.

Just change '.async-hide { opacity: 0 !important}' to use single quotes instead.

Reply:
ff08c4b